go/pkg/bertybot/recipes.go

```diff
@@ -131,10 +130,31 @@ func DelayResponseRecipe(duration time.Duration) Recipe {
 	return recipe
 }
 
-// AutoAcceptIncomingContactRequestRecipe makes the bot "click" on the "accept" button automatically.
-// NOT YET IMPLEMENTED.
+// AutoAcceptIncomingGroupInviteRecipe makes the bot "click" on the "join" button automatically.
 func AutoAcceptIncomingGroupInviteRecipe() Recipe {
-	panic("not implemented.")
+	recipe := map[HandlerType][]Handler{}
+	recipe[IncomingGroupInvitationHandler] = []Handler{
+		func(ctx Context) {
+			err := ctx.ReplyString("I'll join asap !")
```

Comment:
```suggestion
			err := ctx.ReplyString("I'll join asap!")
```

Even better if it can be configurable